.jenkins/common_job_properties.groovy

```diff
@@ -23,11 +23,12 @@ class common_job_properties {
   // Sets common top-level job properties.
   static def setTopLevelJobProperties(def context,
                                       def default_branch = 'master',
-                                      def default_timeout = 100) {
+                                      def default_timeout = 100,
+                                      def repo_link = 'https://github.com/apache/beam') {
 
     // GitHub project.
     context.properties {
-      githubProjectUrl('https://github.com/apache/beam/')
+      githubProjectUrl(repo_link + '/')
```

Comment:
I'm not sure it's even necessary to append the '/'. I believe this just sets the url on the job description page.